openedx/core/djangoapps/credit/tasks.py

```diff
@@ -208,17 +211,19 @@ def _get_proctoring_requirements(course_key):
     # process
     from edx_proctoring.api import get_all_exams_for_course
 
-    requirements = [
-        {
-            'namespace': 'proctored_exam',
-            'name': exam['content_id'],
-            'display_name': exam['exam_name'],
-            'criteria': {},
-        }
-        for exam in get_all_exams_for_course(unicode(course_key))
-        # practice exams do not count towards eligibility
-        if exam['is_proctored'] and exam['is_active'] and not exam['is_practice_exam']
-    ]
+    requirements = []
+    for exam in get_all_exams_for_course(unicode(course_key)):
+        if exam['is_proctored'] and exam['is_active'] and not exam['is_practice_exam']:
```

Comment:
There are few tests failing due to this condition I believe. Please fix those failing tests.